rotors_gazebo_plugins/CMakeLists.txt

```diff
@@ -211,13 +211,9 @@ file(GLOB msgs msgs/*.proto)
 PROTOBUF_GENERATE_CPP(PROTO_SRCS PROTO_HDRS ${msgs})
 
 # Create a shared library of protobuf messages (.so extension on Linux platforms)
-add_library(mav_msgs SHARED ${PROTO_SRCS})
-target_link_libraries(mav_msgs ${PROTOBUF_LIBRARY} gazebo_msgs)
-
-# This causes mav_msgs to be linked with every created library in this file from this
-# point forward.
-# NOTE: This is deprecated, should be using target_link_libraries instead
-link_libraries(mav_msgs)
+add_library(mav_proto_msgs SHARED ${PROTO_SRCS})
```

Comment:
Could you not rename the package? This would need to propagate further than just this cmake file.

Reply:
This doesn't rename a package. It renames the cmake library created for the protobuf msgs